meta-refkit/recipes-core/packagegroups/packagegroup-industrial.bb

```diff
@@ -0,0 +1,14 @@
+SUMMARY = "IoT Reference OS Kit Industrial Robotics profile package groups"
+LICENSE = "MIT"
+PR = "r1"
+
+inherit packagegroup
+
+SUMMARY_${PN} = "IoT Reference OS Kit Industrial Robotics profile"
+RDEPENDS_${PN} = "\
```

Comment:
Do you need a separate packagegroup for development images? Or are the packages the same in both?